src/uct/ib/rdmacm/rdmacm_iface.c

```diff
@@ -407,7 +407,7 @@ uct_rdmacm_iface_process_event(uct_rdmacm_iface_t *iface,
     case RDMA_CM_EVENT_CONNECT_ERROR:
     case RDMA_CM_EVENT_DISCONNECTED:
         /* Server/Client - connection was disconnected */
-        if (status != UCS_ERR_REJECTED) {
+        if (event->event != RDMA_CM_EVENT_UNREACHABLE) {
```

Comment:
next line should also use uct_rdmacm_cm_ep_get_cm(cep)->super.config.failure_level

Reply:
@hoopoepg does this line fix any real bug? IMO both approaches are not nice, so if it dies not, then better to keep it as is

Reply:
> next line should also use uct_rdmacm_cm_ep_get_cm(cep)->super.config.failure_level

done

Reply:
> @hoopoepg does this line fix any real bug? IMO both approaches are not nice, so if it dies not, then better to keep it as is

it shots error message into log, io-demo analyzer interprets it as error and terminates test session